examples/bert/bert_finetune_glue.py

```diff
@@ -202,31 +169,26 @@ def build(self, hp):
 def main(_):
     print(f"Reading input model from {FLAGS.saved_model_input}")
 
-    vocab = []
-    with open(FLAGS.vocab_file, "r") as vocab_file:
-        for line in vocab_file:
-            vocab.append(line.strip())
-    tokenizer = tf_text.BertTokenizer(
-        FLAGS.vocab_file,
-        lower_case=FLAGS.do_lower_case,
-        token_out_type=tf.int32,
+    tokenizer = keras_nlp.tokenizers.WordPieceTokenizer(
```

Comment:
How does the `keras_nlp` tokenizer differ from the `tf_text` one?

Reply:
Essentially it's just a higher level wrapper, with a few added goodies for Keras integration (`tokenize()` and `detokenize()` route through call so things like `model.summary()` will will work as expected, cloning and saving should work).

But the underlying op itself is the same as [FastWordpieceTokenizer](https://www.tensorflow.org/text/api_docs/python/text/FastWordpieceTokenizer).